components/slack/sources/common/base.mjs

```diff
@@ -56,43 +112,52 @@ export default {
           });
           return info.team.name;
         } catch (err) {
-          console.log("Error getting team name, probably need to re-connect the account at pipedream.com/apps", err);
+          console.log(
+            "Error getting team name, probably need to re-connect the account at pipedream.com/apps",
+            err,
+          );
           return id;
         }
       });
     },
     async getLastMessage({
       channel, event_ts,
     }) {
-      return this.maybeCached(`lastMessage:${channel}:${event_ts}`, async () => {
-        const info = await this.slack.sdk().conversations.history({
-          channel,
-          latest: event_ts,
-          limit: 1,
-          inclusive: true,
-        });
+      return this.maybeCached(
+        `lastMessage:${channel}:${event_ts}`,
+        async () => {
+          const info = await this.slack.sdk().conversations.history({
+            channel,
+            latest: event_ts,
+            limit: 1,
+            inclusive: true,
+          });
 
-        return info;
-      });
+          return info;
+        },
+      );
     },
     async getMessage({
       channel, event_ts,
     }) {
-      return await this.maybeCached(`lastMessage:${channel}:${event_ts}`, async () => {
-        const response = await this.slack.sdk().conversations.replies({
-          channel,
-          ts: event_ts,
-          limit: 1,
-        });
+      return await this.maybeCached(
+        `lastMessage:${channel}:${event_ts}`,
+        async () => {
+          const response = await this.slack.sdk().conversations.replies({
+            channel,
+            ts: event_ts,
+            limit: 1,
+          });
```

Comment:
Should avoid using snake case (event_ts).
```
    async getMessage({
      channel, event_ts: ts,
    }) {
      return await this.maybeCached(
        `lastMessage:${channel}:${ts}`,
        async () => {
          const response = await this.slack.sdk().conversations.replies({
            channel,
            ts,
            limit: 1,
          });
```

Reply:
This was also just whitespace, but good catch, renamed it to `ts` upon destructuring. Thanks.